index.js

```diff
@@ -19,4 +21,4 @@ tournament.play().then(() => {
   let report = tournament.report();
   console.log(report);
   process.exit();
-});
+}).catch(err => console.log(err));
```

Comment:
This can seriously burn you if you leave out the catches.  Your server will just timeout.